src/csvToJson.js

```diff
@@ -1,36 +1,19 @@
-/* eslint-disable no-unused-vars */
 import { write, readDir } from '@groceristar/static-data-generator';
-// import { joinPath } from './utils';
-import fs from 'fs';
-import parseCsv from './parseCsv';
 
-function makeFullPath(pathData, filetype) {
-  return ''.concat(pathData[0], pathData[1], filetype); // put pathData + filetype into 1 valid file name string
-}
-
-const generate = (file, data) => {
-  // file => [full directory path, 'filename', 'filetype']
-  const fullPath = makeFullPath(file, '.json');
-
-  console.log('---file writer started---');
-  console.log(fullPath);
-  console.log('---file writer ended---');
+const generateJsonFile = async (fileInfo, data) => {
+  // stringify data with indent
+  const jsonObjects = JSON.stringify(data, null, 2);
 
-  write(fullPath, data);
+  await write(`${fileInfo[0]}/${fileInfo[1]}.json`, jsonObjects);
 };
 
-// @TODO update this method later, when we'll migrate to `write` from generator
-
-// @TODO as this method using "generateJsonFiles" method - it should be updated.
-// or maybe move it into generator file, etc.
-const assign = (file, dataEntries) => {
+const assign = async (fileInfo, dataEntries) => {
   // @TODO add if env.development and use console.log(xxx)
-  const maxEntriesPerFile = 10000;
+  const maxEntriesPerFile = 1000;
```

Comment:
I was thinking, maybe we should set it as method attribute as well, so we can tweak it easily